static/script/devices/mediaplayer/html5.js

```diff
@@ -587,7 +587,7 @@ require.def(
                         self._mediaElement.currentTime = self._sentinelSeekTime;
                     });
                 } else {
-                    this._sentinelSeekTime = currentTime;
+                    this._sentinelSeekTime = undefined;
```

Comment:
Remembered the reason, I think: it was because we'd observed a device where the time would temporarily be reported as the target time for the seek, then skip back to its original position. There can't be a unit test for such a scenario, otherwise this pull request would be breaking the build.

Does that sound right?